hal/src/main/native/athena/HAL.cpp

```diff
@@ -41,6 +43,10 @@ static std::unique_ptr<tGlobal> global;
 static std::unique_ptr<tSysWatchdog> watchdog;
 static uint64_t dsStartTime;
 
+static char roboRioCommentsString[65];
```

Comment:
`65` is used a _lot_ here as a magic number. Extract to constant.

Reply:
Maybe the associated header of this cpp file? Since it'll be included everywhere that uses the constant.

Reply:
Why not `HALBase.h`, along with the declaration of `HAL_GetComments`? Since it seems they're used together from most places (or maybe not? I'm not sure that the HALSIM part includes `HALBase.h`).